packaging/makeself/install-or-update.sh

```diff
@@ -252,23 +252,49 @@ replace_symlink() {
     ln -s "${target}" "${name}"
 }
 
+prep_ssl_dir() {
+  [ -L /opt/netdata/etc/ssl ] && rm /opt/netdata/etc/ssl
+  mkdir -p /opt/netdata/etc/ssl
+}
+
+select_cert_dir() {
+  prefix="/opt/netdata/etc/ssl/certs"
+
+  mkdir -p "${prefix}"
+
+  for bundle in ca-certificates.crt ca-bundle.crt; do
```

Comment:
```suggestion
  for bundle in ca-certificates.crt ca-bundle.crt ca-bundle.pem; do
```

We must use `/etc/ssl/ca-bundle.pem` on openSUSE.